File: EmployeeCrud/EmployeeCrud/Models/Employee.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Threading.Tasks;

namespace EmployeeCrud.Models
{
    public class Employee
    {
        [Key]
        public int EmpId { get; set; }
        public string EmpName { get; set; }
        public string Address { get; set; }
        public string Number { get; set; }
        public double Salary { get; set; }
        public int DesignationId { get; set; }
        [ForeignKey("DesignationId")]
        public Designation Designation { get; set; } 
        public EmpDepTbl Employees { get; set; }
        
    }
}

File: EmployeeCurdFrontEnd/src/app/employee.ts
import { Empdeptbl } from './empdeptbl';
export class Employee {
    id:any;
    empName:any;
    address:any;
    number:any;
    salary:any;
    designationId:any;
    designation:any;
    departmentId:any;
    department:any;
    employees:Empdeptbl; 
    
    
       
    constructor()
    {
        this.id=null;
        this.empName=null;
        this.address=null;
        this.salary=null;
        this.departmentId=null;
        this.department=null;
        this.designationId=null;
        this.designation=null;
        this.employees= this.departmentId;
              
    }

}

